exercises/00_final/precipitation_climatology.py

```diff
@@ -12,34 +12,39 @@
 import regionmask
 
 
-def convert_pr_units(darray):
+def convert_precipitation_units(precipitation_in_kg_per_m_squared_s):
     """
     Convert precipitation units from [kg m-2 s-1] to [mm day-1].
 
     Parameters
     ----------
-    darray : xarray.DataArray
+    precipitation_in_kg_per_m_squared_s : xarray.DataArray
         xarray DataArray containing model precipitation data
 
     Returns
     -------
-    darray : xarray.DataArray
+    precipitation_in_mm_per_day : xarray.DataArray
         the input DataArray with precipitation units modified
     """
     # density 1000 kg m-3 => 1 kg m-2 == 1 mm
     # There are 60*60*24 = 86400 seconds per day
-    darray.data = darray.data * 86400
-    darray.attrs["units"] = "mm/day"
+    precipitation_in_mm_per_day = xr.DataArray(precipitation_in_kg_per_m_squared_s * 86400)
 
-    assert (
-        darray.data.min() >= 0.0
-    ), "There is at least one negative precipitation value"
-    assert darray.data.max() < 2000, "There is a precipitation value/s > 2000 mm/day"
+    precipitation_in_mm_per_day.attrs["units"] = "mm/day"
 
-    return darray
+    if precipitation_in_mm_per_day.data.min() < 0.0:
+        print("There is at least one negative precipitation value")
+    if precipitation_in_mm_per_day.data.max() > 2000:
+        print("There is a precipitation value/s > 2000 mm/day")
 
+    return precipitation_in_mm_per_day
 
-def plot_zonal(data):
+
+# I think it would be good to give more detail here about what the dimensions and
+# content of the input array needs to be, to save the reader having to go through
+# the code and infer it in order to use the method. Would it also be possible to give
+# the input variable a more specific name?
+def plot_zonally_averaged_precipitation(data):
```

Comment:
I *think* it will be the same units as precipitation.

A zonal average is taking an average all the way around the globe at a fixed latitude.